swerve_pico/messaging-library/swerve-module-id.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Defined list of valid message ids and helper function to check and process messages into actions
+ */
+#ifndef _MESSAGING_IDS
+    #define _MESSAGING_IDS
+
+    #define MOVE_IN_DIRECTION 0x80
+
+    #define resultArraySize 6
+    
+    enum messageProcessResult {
+        PAYLOAD_ID_NOT_FOUND,
+        STATE_READ_RESULT_ARRAY,
+        STATE_ZERO_MOTORS
+    };
+
+    float resultArray[resultArraySize] = {0};
```

Comment:
iirc you can't do stuff like this (assigning variables) in a header file but I could be wrong